ynput/ayon-core
# Maya: UDIM texture from Yeti Rig Collector can be searched correctly with absolute path

## Changelog Description
Fixed https://github.com/ynput/ayon-core/issues/713

## Additional info
n/a

## Testing notes:
1. Launch Maya
2. Create Yeti Rig with the absolute texture path
3. Turn Texture mode into UDIM
4. Publish
5. Should be publishing the map as expected.
